src/BenchmarkDotNet/Environments/HostEnvironmentInfo.cs

```diff
@@ -96,7 +96,13 @@ public override IEnumerable<string> ToFormattedString()
                 yield return $"Frequency={ChronometerFrequency}, Resolution={ChronometerResolution.ToString(cultureInfo)}, Timer={HardwareTimerKind.ToString().ToUpper()}";
 
             if (RuntimeInformation.IsNetCore && IsDotNetCliInstalled())
-                yield return $".NET SDK={DotNetSdkVersion.Value}";
+            {
+                // this wonderfull version number contains words like "preview" and ... 5 segments so it can not be parsed by Version.Parse. Example: "5.0.100-preview.8.20362.3"
+                if (DotNetSdkVersion.Value[0] >= '5')
```

Comment:
In November 2025, .NET 10 should be released and this heuristic will be broken. =)
I think it will be better to split the version string by `'.'` and parse the first component. This logic can be encapsulated in a method `int GetMajorVersionComponent(string version)`.